project.json

```diff
@@ -74,6 +74,10 @@
         "{workspaceRoot}/docs/generated"
       ]
     },
+    "check-format:quick": {
+      "cache": true,
```

Comment:
I don't think the inputs on this would be correct by default for caching. Default would be {projectRoot}/**, and project root based globs only glob for files within that projects files which wouldn't include any of our source files, since they belong to other projects